content/3_Silver/Intro_Sorted_Sets.mdx

```diff
@@ -255,24 +349,84 @@ System.out.println(map.lowerKey(3));     // ERROR
 ```
 
 </JavaSection>
+<PySection>
+
+Python does not have built-in sorted maps or sorted sets in its standard
+library. However, with the library `sortedcontainers`, sorted maps in Python can be created by adding a dictionary to
+a sorted set, where each element in the sorted set is a key in the dictionary
+and values can be assigned to the dictionary.
+
+This is the most straightforward implementation, and the ground up
+implementation of a sorted set can be found in the above section.
+
+Since some online judges include additional libraries, an implementation of
+`SortedDict` from the `sortedcontainers` library can be found below. All
+operations shown below are $\mathcal{O}(\log N)$ time, except for an
+$\mathcal{O}(N \log N)$ initialization and $\mathcal{O}(N)$ time to get all
+items, keys, or values.
```

Comment:
ngl this seems kinda a repeat of the last sortedset blurb

Reply:
True, there's some redundancy because I wanted people who might've only read one section (because they already know/don't want to know previous sections) to not be left out of any information. I'll remove the redundancy and assume they have read all previous sections.

Reply:
I also removed
```mdx
Since some online judges include additional libraries, an implementation of
`SortedDict` from the `sortedcontainers` library can be found below.
```
as they are already told about it in the above section, and there is also a warning just in case.

Reply:
yee